packages/flutter/lib/src/foundation/_isolates_io.dart

```diff
@@ -81,7 +81,10 @@ Future<void> _spawn<Q, R>(_IsolateConfiguration<Q, FutureOr<R>> configuration) a
   R result;
   await Timeline.timeSync(
     '${configuration.debugLabel}',
-    () async { result = await configuration.apply(); },
+    () async { 
```

Comment:
nit: There's an extra whitespace at the end of this line that the analyzer is complaining about.